applications/machine_learning/sysbuild.cmake

```diff
@@ -6,6 +6,9 @@
 
 if (${NORMALIZED_BOARD_TARGET} STREQUAL "nrf54h20dk_nrf54h20_cpuapp")
   list(LENGTH SHIELD SHIELD_NUM)
+  list(LENGTH machine_learning_SHIELD APP_SHIELD_NUM)
+  math(EXPR SHIELD_NUM "${SHIELD_NUM} + ${APP_SHIELD_NUM}")
```

Comment:
Shouldn't we also include remote_SHIELD?

Reply:
I don't think so. Remote shield is not mandatory when shield for app is necessary